src/__mocks__/graceful-fs.js

```diff
@@ -8,17 +8,20 @@
  */
 'use strict';
 
-var fs = jest.genMockFromModule('fs');
+const fs = jest.genMockFromModule('fs');
+const noop = () => {};
 
-function asyncCallback(callback) {
-  return function() {
-    setImmediate(() => callback.apply(this, arguments));
-  };
-}
+const asyncCallback = (cb) => function() {
```

Comment:
the code style is not to use parens for arrow functions with one param that is an identifier :)